src/pages/index.astro

```diff
@@ -209,7 +209,7 @@ const description = t("site.description");
 
 <style>
   .section {
-    border-bottom: 1px solid var(--sl-color-gray-2);
+    border-block-end: 1px solid var(--sl-color-gray-2);
```

Comment:
The rest of the changes from here on out are converting the old syntax to logical properties syntax.